package dev.weazyexe.reko.data.error

/**
 * Exception can be thrown while recognizing via SkyBiometry
 * and there are no faces
 */
class FacesNotFoundException: Throwable()